DEVELOPING.md

```diff
@@ -26,6 +26,28 @@ To run the examples you first need to start the dev server:
 
 Then, load <http://localhost:8080/> in your browser.
 
+## Linking Package
+
+The `ol` package is published from the `build/ol` folder of the `openlayers` repo. 
+
+After you've cloned the `openlayers` repo locally run the `npm build-package` to prepare the build then use the `npm link` command to connect it your project. 
+
+Below is an example of how to build and link it to "sample-project"
+
+
+    $ cd openlayers
+    $ npm build-package
```

Comment:
Shouldn't this be `npm run build-package`?

Reply:
Good catch - I just submitted a PR with this fix https://github.com/openlayers/openlayers/pull/13231